packages/state-manager/src/sagas/messages/checkForMessages/checkForMessages.saga.ts

```diff
@@ -1,20 +1,27 @@
 import { put, select } from 'typed-redux-saga'
 import {
+  currentPublicChannelMessagesBase,
   missingChannelMessages
 } from '../messages.selectors'
-import { PayloadAction } from '@reduxjs/toolkit'
+import { EntityId, PayloadAction } from '@reduxjs/toolkit'
 import { messagesActions } from '../messages.slice'
 import { currentCommunity } from '../../communities/communities.selectors'
 import { currentIdentity } from '../../identity/identity.selectors'
 
 export function* checkForMessagesSaga(action: PayloadAction<ReturnType<typeof messagesActions.responseSendMessagesIds>['payload']>): Generator {
   const { ids, channelAddress } = action.payload
+  let messagesIds: string[] = ids
+
+  if(messagesIds.length === 0){
```

Comment:
I remember we agreed that changes in this file are not necessary as the problem lies somewhere else?

Reply:
leftover